package.json

```
@@ -3,7 +3,7 @@
   "private": true,
   "version": "2.1.0",
   "scripts": {
-    "version": "changeset version && PACKAGE_VERSION=$(node -p \"require('./package.json').version\") && echo \"module Evervault\n\tVERSION = \\\"$PACKAGE_VERSION\\\"\\n\tEV_VERSION = {\\\"prime256v1\\\" => \\\"NOC\\\", \\\"secp256k1\\\" => \\\"DUB\\\"}\nend\" > ./lib/evervault/version.rb"
+    "version": "changeset version && PACKAGE_VERSION=$(node -p \"require('./package.json').version\") && echo \"module Evervault\n\tVERSION = \\\"$PACKAGE_VERSION\\\"\\n\tEV_VERSION = {\\\"prime256v1\\\" => \\\"LCY\\\", \\\"secp256k1\\\" => \\\"BRU\\\"}\nend\" > ./lib/evervault/version.rb"
```

Here you could just use sed instead of updating the whole file

```suggestion
    "version": "changeset version && PACKAGE_VERSION=$(node -p \"require('./package.json').version\") && sed -i "s/^ *VERSION *= *\"[^\"]*\"/VERSION = \"$PACKAGE_VERSION\"/" lib/evervault/version.rb"
```